p2p/src/peer_manager/peerdb/address_data/mod.rs

```diff
@@ -47,30 +46,47 @@ pub enum AddressState {
     },
 }
 
+#[derive(Copy, Clone, Debug)]
+// Update `ALL_TRANSITIONS` if a new transition is added!
```

Comment:
[`EnumIter` from the `strum` crate](https://docs.rs/strum_macros/0.24.3/strum_macros/derive.EnumIter.html) can be used instead.

Reply:
Right, thanks. It's only for tests so I think we can keep it.